main.tf

```diff
@@ -292,3 +285,24 @@ data "cloudflare_zones" "domain" {
     status      = "active"
   }
 }
+
+locals {
+  table_names = {
+    stg  = "sildisco_dev_user-log"
+    prod = "sildisco_prod_user-log"
+  }
+}
+
+resource "aws_dynamodb_table" "logger" {
```

Comment:
Do we want a `tags` entry on this?

Reply:
The only one it still needs is `name` or `Name` but there's some unresolved discussion about whether it should be capitalized, and some question in my mind about the specific content of that tag.